src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/TraceLoggingDataCollector.cs

```diff
@@ -34,7 +34,7 @@ private TraceLoggingDataCollector()
         /// Marks the start of a non-blittable array or enumerable.
         /// </summary>
         /// <returns>Bookmark to be passed to EndBufferedArray.</returns>
-        public int BeginBufferedArray()
+        public static int BeginBufferedArray()
```

Comment:
If all of the methods are now static, can the private ctor and Instance static above be removed, and the type be made static?

Reply:
It's not fully static, see https://github.com/marek-safar/runtime/blob/spc/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/TraceLoggingDataCollector.cs#L58

Reply:
Is the AddGroup method actually used for anything useful?

Reply:
I don't know, there is a comment about some correctness verification on the method and there is `#if ES_BUILD_STANDALONE` in the file so not sure if that's just some legacy or not

Reply:
I think it should be pretty safe to ignore this comment and delete this. I do not see what kind of correctness verification can this be used for.

Reply:
Done